dash/include/dash/GlobAsyncRef.h

```diff
@@ -157,6 +157,30 @@ class GlobAsyncRef
   {
     return _is_local;
   }
+  
+  /**
+   * Get a global ref to a member of a certain type at the
+   * specified offset
+   */
+  template<typename MEMTYPE>
+  GlobAsyncRef<MEMTYPE> member(size_t offs) const {
+    dart_gptr_t dartptr = _gptr;
+    DASH_ASSERT_RETURNS(
+      dart_gptr_incaddr(&dartptr, offs),
+      DART_OK);
+    GlobConstPtr<MEMTYPE> gptr(dartptr);
+    return GlobAsyncRef<MEMTYPE>(gptr);
```

Comment:
This would trigger a new `get` even though the current `GlobAsyncRef` may already have cached the data, potentially leading to incoherency. How about a private ctor `GlobAsyncRef(dart_gptr_t, MEMTYPE*)`?

Also: the intermediate `GlobConstPtr` is not required, right?